Smartling.API/Authentication/AuthApiUser.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace Smartling.Api.Authentication
+{
+    public class AuthApiUser : AuthApiClient
+    {
+        protected override string AuthApiV2Authenticate { get { return "/auth-api/v2/authenticate/user"; } }
+
+        public AuthApiUser(string userIdentifier, string userSecret) : base(userIdentifier, userSecret)
+        {
+        }
```

Comment:
I don't know C#, just a question. Why did you override define this method? Seems like it has an empty body.